pkg/kclient/kclient.go

```diff
@@ -0,0 +1,53 @@
+package kclient
+
+import (
+	"errors"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// errorMsg is the message for user when invalid configuration error occurs
+const errorMsg = `
+Please ensure you have an active kubernetes context to your cluster. 
+Consult your Kubernetes distribution's documentation for more details
+`
+
+// Client is a collection of fields used for client configuration and interaction
+type Client struct {
+	KubeClient       kubernetes.Interface
+	KubeConfig       clientcmd.ClientConfig
+	KubeClientConfig *rest.Config
+	Namespace        string
+}
+
+// New creates a new client
+func New(skipConnectionCheck bool) (*Client, error) {
```

Comment:
Do we need `skipConnectionCheck`? It is not used anywhere in the function.

Reply:
Ah, good catch @kadel, that can be removed as it's not being used right now (and the kclient in our kdo-poc branch never used it)